airbyte-api/src/main/openapi/config.yaml

```diff
@@ -121,6 +121,48 @@ paths:
           description: Workspace not found
         "422":
           $ref: "#/components/responses/InvalidInput"
+  /v1/sources/create:
```

Comment:
what is calling this?

Reply:
Just to make sure we are fully aligned on the terminology:

does it mean we will call source-connector: source?
does it mean we will call destination-connector: destination?

Challenging it: why don't we call it connector (with an enum type to distinguish between source and destination)?